src/DendroDocs.Tool/Analyzers/InvocationsAnalyzer.cs

```diff
@@ -71,23 +71,14 @@ public override void VisitInvocationExpression(InvocationExpressionSyntax node)
             return;
         }
 
-        var containingType = semanticModel.GetSymbolInfo(expression).Symbol?.ContainingSymbol.ToDisplayString();
-        if (containingType == null)
-        {
-            containingType = semanticModel.GetSymbolInfo(expression).CandidateSymbols.FirstOrDefault()?.ContainingSymbol.ToDisplayString();
-        }
+        var containingType = semanticModel.GetSymbolInfo(expression).Symbol?.ContainingSymbol.ToDisplayString() ?? semanticModel.GetSymbolInfo(expression).CandidateSymbols.FirstOrDefault()?.ContainingSymbol.ToDisplayString() ?? string.Empty;
 
-        var methodName = string.Empty;
-
-        switch (node.Expression)
+        var methodName = node.Expression switch
```

Comment:
This looks like only applying modern syntax? I like it, but as it has nothing to do with the other change in this file, split this in a separate commit?